src/ol/style/icon.js

```diff
@@ -260,6 +260,16 @@ ol.style.Icon.prototype.getAnchor = function() {
 
 
 /**
+ * Get the icon color.
+ * @return {ol.Color|string} Color.
+ * @api
+ */
+ol.style.Icon.prototype.getColor = function() {
+  return this.color_;
```

Comment:
but is it same as in[ fill.js](/openlayers/ol3/blob/v3.19.1/src/ol/style/fill.js#L51) and [stroke.js](/openlayers/ol3/blob/v3.19.1/src/ol/style/stroke.js#L88) ?